src/types/projectTypes.ts

```diff
@@ -31,3 +31,8 @@ export interface TagData {
   id: number
   name: string
 }
+
+export interface FullProject extends Omit<ProjectData, "sequence"> {
```

Comment:
I think I'm a little confused about this interface there too. Let's talk about but seems maybe like we're doing a lot of work here to omit a couple of fields from the sequence interface

Reply:
keeping this because you were a silly sausage